Scripts/public-api-diff/Sources/Helpers/Models/SDKDump+Element+Definition.swift

```diff
@@ -1,8 +1,7 @@
 //
-//  File.swift
-//  
+// Copyright (c) 2024 Adyen N.V.
 //
-//  Created by Alexander Guretzki on 15/07/2024.
```

Comment:
The latest develop excludes these files from being formatted - would you mind reverting the changes on public-api-diff?